qemu/tests/qemu_img_check_data_integrity.py

```diff
@@ -89,3 +92,17 @@ def run_iozone_background(vm):
         session = vm.wait_for_login()
         session.cmd(params["rm_testfile_cmd"] % iozone_testfile)
     session.close()
+    vm.destroy()
```

Comment:
hi @hellohellenmao , if we don't quit vm gracefully(e.g. gracefully=False), is that much more likely to hit the 'leaked clusters'?

Reply:
Yeah, it should be. But I do not think we need it. The leaked clusteres should be created while `kill_vm_process` in previous steps, but not in this step.